docs/reference/search/search-your-data/cohere-es.asciidoc

```diff
@@ -0,0 +1,370 @@
+[[cohere-es]]
+=== Tutorial: Using Cohere with {es}
+++++
+<titleabbrev>Using Cohere with {es}</titleabbrev>
+++++
+
+The instructions in this tutorial shows you how to compute embeddings with
+Cohere using the {infer} API and store them for efficient vector or hybrid
+search in {es}. This tutorial will use the Python {es} client to perform the
+operations.
+
+You'll learn how to:
+* create an {infer} endpoint for text embedding using the Cohere service,
```

Comment:
Should the `*` render in a specific way?

Looks like this in my browser when I view the preview

![image](https://github.com/elastic/elasticsearch/assets/56361221/bfb12836-3ef5-4cba-a63b-44d4c79051a9)